src/VisualStudio/Core/Def/InheritanceMargin/InheritanceGlyphManager.cs

```diff
@@ -43,7 +43,11 @@ internal sealed partial class InheritanceGlyphManager : IDisposable
     private readonly IEditorFormatMap _editorFormatMap;
     private readonly IAsynchronousOperationListener _listener;
     private readonly Canvas _glyphsContainer;
-    private readonly SimpleIntervalTree<GlyphData, GlyphDataIntrospector> _glyphDataTree;
+
+    /// <summary>
+    /// Mutable.  Must only be accessed from the UI thread.
+    /// </summary>
+    private SimpleBinaryIntervalTree<GlyphData, GlyphDataIntrospector> _glyphDataTree;
```

Comment:
tree construction at end.  this will move to the FlatArrayIntervalTree in next pr.